src/cfnlint/__init__.py

```diff
@@ -18,16 +18,19 @@
 import sys
 import os
 import re
-from copy import deepcopy
+from copy import deepcopy, copy
 from datetime import datetime
 import six
 from yaml.parser import ParserError
 import cfnlint.helpers
+import cfnlint.conditions
 from cfnlint.transform import Transform
 from cfnlint.decode.node import TemplateAttributeError
 
 LOGGER = logging.getLogger(__name__)
 
+# pylint: disable=too-many-lines
```

Comment:
Perhaps just disable this linter rule for the entire project?

Reply:
yea possibly.  Was kind of debating if we should start splitting out some of those larger classes into their own files.  Then we could remove this one.